proto/resource_manager.proto

```diff
@@ -148,7 +150,9 @@ enum GroupMode {
 message ResourceGroup {
   string name = 1;
   GroupMode mode = 2;
+  // Used in RU mode, group settings with WRU/RRU etc resource abstract unit.
   GroupRequestUnitSettings r_u_settings = 3;
```

Comment:
how about ru_mode_setting and raw_mode_setting

Reply:
How about now? `mode` already a field so I not used it.